assets/prototype/domain/eventEditor/datetimes/datesList/AddNewDateModal.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import DateForm from '../dateForm/DateForm';
-import FormModal from '../../../../application/ui/components/forms/FormModal';
+import FormModal from '../../../../application/ui/components/forms/FormModal/FormModal';
```

Comment:
@tn3rb should I create `index.tsx` under the `FormModal` and move the content from `FormModal.js` into that file?